### PR TITLE
Updates to the README instructions

### PR DESCRIPTION
**Pull request by lrpablo**
Updated main README.md and exercise 1 & 2 READMEs with DevCloud-via-SSH instructions. If approved, I'll update the remaining exercise READMEs.

### CODE REVIEW

**Review by DuncanMcBain — COMMENTED** [review]
This looks good but I was wondering if we could save ourselves the repetition and put the (generic) instructions in a README in the Code_Exercises directory, then if there are specific tweaks leave them in the individual directories. Does that make sense?

File: README.md [hunk 1]
```diff
@@ -258,10 +258,37 @@ Once you have completed any given exercise make sure to compare your implementat
...
 
-SYCL and the SYCL logo are trademarks of the Khronos Group Inc.
+## Connecting to DevCloud via SSH
+
+* Start by creating an [Intel DevCloud account][intel-devcloud] account if you do not already have one and login in.
+* Initialize the SSH configuration by clicking on [Automated Configuration](https://devcloud.intel.com/oneapi/documentation/connect-with-ssh-linux-macos/) and follow the instructions to setup the SSH configuration file.
+* SSH into DevCloud (```ssh devcloud```)
+
+You are now ready to start with the first [lesson][lesson-1-slides]. Enjoy !
+
```
Comment (rodburns) [inline README.md:268]: I would still like to see instructions for accessing a Terminal through the Jupyter notebook access as this is the simplest for some people. There is a GUI for editing files and a full terminal window as well. One issue is that this seems to be moved around sometimes, but currently it is on this page https://console.cloud.intel.com/training. Maybe this is too hard to keep up to date though?

**Review by rodburns — APPROVED** [review]
Do you think we

**Review by DuncanMcBain — APPROVED** [review]
If we can fix the newline I'll merge :+1:

**Pull request merged**